.travis.yml

```diff
@@ -43,6 +43,10 @@ matrix:
           dist: trusty
           compiler: clang
           env: CONFIG_OPTS="--strict-warnings -D__NO_STRING_INLINES no-deprecated" BUILDONLY="yes"
+        - os: linux
+          dist: trusty
+          compiler: clang
+          env: CONFIG_OPTS="--strict-warnings -DFIPS_MODE -D__NO_STRING_INLINES no-deprecated"
```

Comment:
I wonder if this should be a BUILDONLY="yes" environment...

Reply:
Some tests use FIPS_MODE currently and it would be nice to run them.

I think there is a better way to do this for testing which shouldn't require double compilation.